index.html

```diff
@@ -13297,9 +13297,9 @@ <h2>Excluding Elements from the Accessibility Tree</h2>
 	  <h2>Including Elements in the Accessibility Tree</h2>
           <p>If not excluded from the accessibility tree per the rules above in <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a>, user agents MUST provide an <a>accessible object</a> in the <a>accessibility tree</a> for <abbr title="Document Object Model">DOM</abbr> <a>elements</a> that meet any of the following criteria:</p>
           <ul>
-            <li>Elements that are not <a>hidden</a> and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>:
+            <li>Elements that are not <a>hidden</a> and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>, including:
               <ul>
-                <li>Elements that are <a>focusable</a> even if the element or one of its ancestor elements has its <sref>aria-hidden</sref> attribute set to <code>true</code>.</li>
+                <li>Elements that are currently <a>focused</a>, even if the element or one of its ancestor elements has its <sref>aria-hidden</sref> attribute set to <code>true</code>.</li>
```

Comment:
We should take out the definition link for now.

The definition of "focusable" (and/or "focused") may be added some time in future in https://github.com/w3c/aria-common/pull/39, and at that time, we can decide which of the 61 instances of "focusable" and 17 instances of "focused" in the ARIA spec should be linked to the definition. 

Since we haven't needed that definition before, we probably shouldn't add it right before 1.2.

```suggestion
                <li>Elements that are currently focused, even if the element or one of its ancestor elements has its <sref>aria-hidden</sref> attribute set to <code>true</code>.</li>
```